ajax/libs/highland/2.10.5/highland.js

```diff
@@ -0,0 +1,8418 @@
+(function(f){if(typeof exports==="object"&&typeof module!=="undefined"){module.exports=f()}else if(typeof define==="function"&&define.amd){define([],f)}else{var g;if(typeof window!=="undefined"){g=window}else if(typeof global!=="undefined"){g=global}else if(typeof self!=="undefined"){g=self}else{g=this}g.highland = f()}})(function(){var define,module,exports;return (function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module '"+o+"'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({1:[function(require,module,exports){
+(function (process,global){
+/**
+ * Highland: the high-level streams library
+ *
+ * Highland may be freely distributed under the Apache 2.0 license.
+ * http://github.com/caolan/highland
+ * Copyright (c) Caolan McMahon
+ *
+ */
+
+
+var inherits = require('util').inherits;
+var EventEmitter = require('events').EventEmitter;
+var Decoder = require('string_decoder').StringDecoder;
+
+var Queue = require('./queue');
```

Comment:
My diff report some difference between npm highland@2.10.5
https://unpkg.com/highland@2.10.5/dist/highland.js
and here.
Would you check these file again?